Document describe.one usage (#4957)
chef/inspec: docs/dsl_inspec.md

end
```

#### `describe.one` usage notes

> * InSpec considers a `describe.one` block as passing if a `describe` nested within has all assertions passing. That is, it needs an entire `describe` block to pass and not just single assertions.

Reviewer: "not just a single assertion"

> * InSpec considers a `describe.one` block as passing if a `describe` nested within has all assertions passing. That is, it needs an entire `describe` block to pass and not just a single assertion.

Reviewer 2: It may be the result of too little coffee, but I found this and some of the following sentences hard to follow on first reading.  I think some of it might be the result of the tense of this sentence:

```suggestion
* A `describe.one` block passes if one of its nested `describe` blocks have all assertions passing. That is, it needs an entire `describe` block to pass and not just a single assertion.
```

Author: Thanks Steven, that is a neater way to write it. I think `has all assertions` is correct because a singular block is the subject.

Reviewer 2: @james-stocks Yes, sorry, I was playing around with a few different ideas and let that error creep in. `has` is definitely correct here.
